packages/theming/win32-theme/package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
     "@fluentui-react-native/scripts": "^0.1.1",
+    "@office-iss/react-native-win32": ">=0.68.0 <0.68.6",
```

Comment:
Could we make the server match the other ones we have for react-native-win32? this format looks different to me

Reply:
Also can we add it to peer dependencies below.. and maybe add the "core-win32" capability to "rex-kit" if it exists?

Reply:
? Where are you looking at where this is different

Reply:
Talked offline, I was wrong, nvm.